app/models/user.rb

```diff
@@ -9,4 +9,40 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :requested_friendships, foreign_key: 'requester_id', class_name: 'Friendship'
+  has_many :received_friendships, foreign_key: 'receiver_id', class_name: 'Friendship'
+
+  #  has_many :friendships
+  #  has_many :inverse_friendships, :class_name => "Friendship", :foreign_key => "friend_id"
```

Comment:
- [ ] Let's remove commented out code, you can save it to be revisited in another branch or past commit. Keep your code base clean :-)